### PR TITLE
Check wac-allow header in case it disallows write

### PR DESCRIPTION
**Pull request by timbl**
Addressing

https://github.com/linkeddata/rdflib.js/issues/332

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "rdflib",
   "description": "an RDF library for node.js. Suitable for client and server side.",
-  "version": "0.21.1",
+  "version": "0.21.1-1",
```
Comment (megoth) [inline package.json]: Unless you've published a prerelease to NPM, I don't think we should merge this.

File: src/update-manager.js [hunk 2]
```diff
@@ -100,10 +100,11 @@ class UpdateManager {
     var definitive = false
     var requests = kb.each(undefined, this.ns.link('requestedURI'), docpart(uri))
 
-    // Hack for the moment @@@@ 2016-02-12
+    /* "Hack for the moment" @@@@ 2016-02-12 - Dimitri added (why?); timbl removed 2019-08
```
Comment (megoth) [inline src/update-manager.js]: @dmitrizagidulin could we perhaps remove this hack altogether?

Comment (megoth) [reply src/update-manager.js]: Always nice to remove code if we can ^_^

Comment (dmitrizagidulin) [reply src/update-manager.js]: I'm sorry, I don't remember what the context for that was :) Feel free to remove.

Comment (megoth) [reply src/update-manager.js]: Thank you, I'm removing this in the latest commit ^_^

**Pull request merged**